### ISSUE TITLE
[Bug] Cannot save LAND parcel with new objects

### ISSUE THREAD

**Issue by HAZARDU5** [decentraland/feedback#7]
![screenshot-2017-12-02-09 17](https://user-images.githubusercontent.com/2382278/33501694-5a0a2f00-d742-11e7-936b-58b485f31804.jpg)
I'm having some issues saving LAND parcels when editing groups 4x4 in size.

The following seem to be affected:

5, 5
6, 5
7, 5
8, 5
5, 6
6, 6
7, 6
8, 6
5, 7
6, 7
7, 7
8, 7
5, 8
6, 8
7, 8
8, 8

**Comment by Alonzo-Coeus** [comment]
i have the same problem

**Comment by HAZARDU5** [comment]
Yep just tried this with some new parcels and still getting the same issue. Error is slightly different now:
```
Uncaught TypeError: Cannot read property 'getHostNode' of null
    at Object.getHostNode (aframe-inspector.js:57586)
```
![screenshot 2017-12-10 22 10 56](https://user-images.githubusercontent.com/2382278/33803509-0c8ea466-ddf7-11e7-9820-86702a70639f.png)
